src/CMakeLists.txt

```diff
@@ -251,7 +251,10 @@ find_package(PkgConfig)
 pkg_check_modules(SQLITE3 REQUIRED IMPORTED_TARGET sqlite3)
 target_link_libraries(migraphx PRIVATE PkgConfig::SQLITE3)
 
-find_package(msgpack REQUIRED)
+find_package(msgpackc-cxx REQUIRED)
```

Comment:
`REQUIRED` needs to be removed on the first `find_package`.

Reply:
Ah thx for the catch

Reply:
Updated